### PR TITLE
fix: 🧪 update yarn without integrity

### PR DESCRIPTION
**Pull request by Beraliv**
Related to #657

### CODE REVIEW

File: test/fixtures/update-yarn-without-integrity/yarn.lock [hunk 1]
```diff
@@ -0,0 +1,508 @@
...
+  resolved "https://registry.yarnpkg.com/browserslist/-/browserslist-4.9.1.tgz#01ffb9ca31a1aef7678128fc6a2253316aa7287c"
+  integrity sha512-Q0DnKq20End3raFulq6Vfp1ecB9fh8yUNV55s8sekaDDeqBaCtWlRHCUdaWyUeSSBJM7IbM6HcsyaeYqgeDhnw==
+  dependencies:
+    caniuse-lite "^1.0.30001030"
+    electron-to-chromium "^1.3.363"
+    node-releases "^1.1.50"
+
+caniuse-lite@^1.0.30000981, caniuse-lite@^1.0.30001020, caniuse-lite@^1.0.30001030:
+  version "1.0.30001035"
+  resolved "https://registry.yarnpkg.com/caniuse-lite/-/caniuse-lite-1.0.30001035.tgz#2bb53b8aa4716b2ed08e088d4dc816a5fe089a1e"
```
Comment (Beraliv) [inline test/fixtures/update-yarn-without-integrity/yarn.lock:46]: added packages without integrity line

File: update-db.js [hunk 2]
```diff
@@ -151,12 +151,14 @@ function updateYarnLockfile(lock, latest) {
           /resolved "[^"]+"/,
           'resolved "' + latest.dist.tarball + '"'
         )
-        lines[3] = latest.dist.integrity
-          ? lines[3].replace(
-              /integrity .+/,
-              'integrity ' + latest.dist.integrity
-            )
-          : ''
+        if (lines.length === 4) {
```
Comment (Beraliv) [inline update-db.js:154]: Added fix checking the length of the lines

File: test/update-db/index.test.js [hunk 3]
```diff
@@ -209,6 +209,16 @@ test('updates caniuse-lite for yarn', async () => {
   checkYarnLockfile(dir)
 })
 
+test('updates caniuse-lite for yarn without integrity', async () => {
+  let dir = await chdir(
+    'update-yarn-without-integrity',
```
Comment (Beraliv) [inline test/update-db/index.test.js:214]: Added the test before the fix

- [x] Checked that it's failed
- [x] Check that it's resolved after the fix

**Comment by ai** [comment]
Thanks. Released in 4.19.

**Pull request merged**